timescales/templates/ts-base.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+
+{% block title %}{% endblock %}
+
+{% block content %}
+    <div class="container-fluid">
+        <div class="geo-content">
+            {% block geo-content %}
+            {% endblock %}
+        </div>
+    </div>
+{% endblock %}
```

Comment:
As discussed, may become redundant in the future but we can keep it for now

Reply:
I don't see why, as the layout should remain consistent. We'll see.